docs/operator-manual/disaster_recovery.md

```diff
@@ -25,4 +25,4 @@ docker run -i -v ~/.kube:/home/argocd/.kube --rm quay.io/argoproj/argocd:$VERSIO
 ```
 
 !!! note
-    If you are running Argo CD on a namespace different than default remember to pass the namespace parameter (-n <namespace>). 'argocd admin export' will not fail if you run it in the wrong namespace.
+    If you are running Argo CD on a namespace different than default remember to pass the namespace parameter (-n <namespace>) so that 'argocd admin export' will not fail (it will look for the `argocd-cm` configMap).
```

Comment:
```suggestion
    If you are running Argo CD on a namespace different than default remember to pass the namespace parameter (-n <namespace>) so that 'argocd admin export' command can pass as this command looks for the resources in the default namespace (`argocd`).
```